stdlib/builtin/nil_class.rbs

```diff
@@ -32,6 +32,13 @@ class NilClass < Object
   # ```
   def to_h: () -> ::Hash[untyped, untyped]
 
+  # Always returns zero.
+  # 
+  # ```ruby
+  # nil.to_i   #=> 0.0
```

Comment:
nil.to_i returns `0`, not `0.0`. Could you update this comment?

Reply:
Thank you for the review. Fixed it. https://github.com/ruby/ruby-signature/pull/66/commits/cdfe4e5709b3d58ec7c0905eec18cc55bf6c44ee